mysite/notes/views.py

```diff
@@ -0,0 +1,52 @@
+from django.http import HttpResponse
+from notes.models import MyNotes
+from django.shortcuts import render
+from django.http import HttpResponse, HttpResponseRedirect
+from django.urls import reverse
+from django.views.generic import ListView, UpdateView, DeleteView
+from notes.forms import NoteForm
+
+
+def addnote(request):
+    if request.method == 'POST':
+        form = NoteForm(request.POST)
+        error = None
+        if form.is_valid():
+            form.save()
+            return HttpResponseRedirect(reverse('notes:viewnotes'))
+        else:
+            error = "Please add correct information in all fields!!"
```

Comment:
Do not use this message, this is not helpful.

Reply:
Sure.